src/features/common/components/Home/Home.js

```diff
@@ -1,15 +1,18 @@
 import { useDonationsAndPetitions } from 'features/common';
+import { useAuth } from 'features/auth';
 import UserInfo from './UserInfo';
 import Donations from './Donations';
 import Petitions from './Petitions';
 
 const Home = () => {
   const { donationsStatus, donationsData, petitionsStatus, petitionsData } =
     useDonationsAndPetitions();
+  const { user } = useAuth();
+  const missingUserInfo = !user.phone || !user.about_me;
```

Comment:
Try to add a verb on the boolean variables, something like
```suggestion
  const hasMissingInfo = !user.phone || !user.about_me;
```